docs/api.rst

```diff
@@ -5,6 +5,8 @@ API Reference
 Basic Exploration
 *****************
 
+.. automethod:: api.client.gro_client.GroClient.find_data_series
```

Comment:
These appear in whatever order you put them in in this file, and similar functions are grouped together, in a progression, like: `lookup()`, `search()`, `search_and_lookup()`

Following that that same idea, `find_data_series()` should come after `get_data_series()`, since it's a combo of `search()` and `get_data_series()`

Reply:
yup I saw they appear in the same order.   I like the thematic groups. within each group, I think it makes sense to go from simplest to hardest which is why I put this first.

Reply:
I think there's also a flow from "entity" into "data series" though, which this breaks, since this sandwiches entity methods in-between the two data series methods

Reply:
hmm I was thinking of the flow being more like increasing complexity for the user, i.e. "I'm feeling lucky" is the first thing. THen when you get more sophisticated you start  looking at  lower level details. But that doesn't quite fit with lookup being before search, which is more like introducing more complicated concepts.  Ditto for data retrieval and pandas. So we can do taht in a separate pr